### PR TITLE
fix(FOROME-397) fix filter tags section variants with notes option

### PR DESCRIPTION
**Pull request by evgeniy-chernomortsev**

### CODE REVIEW

File: src/pages/ws/ui/control-panel-filter-tags.tsx [hunk 1]
```diff
@@ -69,6 +70,13 @@ const ModalElement = observer(({ close, title }: ModalProps) => {
       datasetStore.addZone(['_tags', zoneStore.selectedTags])
     }
 
+    if (zoneStore.isModeWithNotes) {
+      datasetStore.addZone([
+        '_tags',
+        [...toJS(zoneStore.selectedTags), '_note'],
```
Comment (AndreyUstyumenko) [inline src/pages/ws/ui/control-panel-filter-tags.tsx:76]: no need to use toJS.

Comment (evgeniy-chernomortsev) [reply src/pages/ws/ui/control-panel-filter-tags.tsx]: +

**Review by AndreyUstyumenko — APPROVED** [review]

**Review by Lapk1n — CHANGES_REQUESTED** [review]
1. Open PGP3140_wgs_panel
2. Choose variants with notes only
3. Main table displays 7 variants
The 4th variant doesn't have notes inside

**Comment by evgeniy-chernomortsev** [comment]
> 1. Open PGP3140_wgs_panel
> 2. Choose variants with notes only
> 3. Main table displays 7 variants
>    The 4th variant doesn't have notes inside

these 7 variants we are getting from back-end putting string "_note" to api request (not filtering list manually), so maybe we should check this case on back-end at first?

**Review by Lapk1n — APPROVED** [review]

**Review by YaroslavTrefilovBraveDevelopers — APPROVED** [review]

**Pull request merged**